package/MDAnalysis/analysis/lineardensity.py

```diff
@@ -262,11 +323,10 @@ def _single_frame(self):
                 f"{self.grouping} is not a valid value for grouping."
             )
 
-        self.totalmass = np.sum(self.masses)
+        # self.totalmass = np.sum(self.masses)
```

Comment:
`totalmass` is not used anywhere in the code. It's also not documented.

Furthermore, it's not clear what it should contain when we have updating ag's.

Therefore, let's remove it as part of this PR.